modules/Hoot/ui/modals/deleteFavoriteOpts.js

```diff
@@ -0,0 +1,180 @@
+import FormFactory   from '../../tools/formFactory';
+import AdvancedOpts  from '../../ui/sidebar/advancedOpts';
+import Sidebar       from '../../ui/sidebar/sidebar';
+import { deleteFavoriteOpts} from '../../config/domMetadata';
+import _find    from 'lodash-es/find';
+import LayerConflate from '../sidebar/layerConflate';
+
+/**
+ * Form that allows user to import datasets into hoot
+ *
+ * @param translations - All translations from database
+ * @constructor
+ */
+export default class DeleteFavoriteOpts {
+    constructor( opts ) {
+        this.favorites  = opts;
+    }
+
+    render() {
+
+        this.deleteOpts = deleteFavoriteOpts.call( this );
+
+        let optCheck = [];
```

Comment:
Jus a style suggestion, but rather than make a list, loop, push to list, could just be opportunity for a .map
`optCheck=this.favorites.map(a => a.name)`